qiita_ware/ebi.py

```diff
@@ -922,13 +927,13 @@ def generate_demultiplexed_fastq(self, rewrite_fastq=False, mtime=None):
             - The demux file couldn't be read
             - All samples are removed
         """
-        ppd = self.preprocessed_data
+        ar = self.artifact
 
         dir_not_exists = not isdir(self.full_ebi_dir)
         if dir_not_exists or rewrite_fastq:
             makedirs(self.full_ebi_dir)
 
-            demux = [path for _, path, ftype in ppd.get_filepaths()
+            demux = [path for _, path, ftype in ar.get_filepaths()
                      if ftype == 'preprocessed_demux'][0]
```

Comment:
Any reason why only the first `demux` path is chosen? (A comment would be nice).